pkg/oonf_api/Makefile.include

```diff
@@ -1,5 +1,5 @@
 INCLUDES += -I$(PKGDIRBASE)/oonf_api/src-api
 
-ifeq ($(shell uname -s),Darwin)
-	CFLAGS += -Wno-reserved-id-macro -Wno-keyword-macro
+ifeq ($(TOOLCHAIN), llvm)
+    CFLAGS += -Wno-reserved-id-macro -Wno-keyword-macro -Wno-parentheses-equality
```

Comment:
@gebart I think `-Wno-reserved-id-macro` is no needed, but `-Wno-parentheses-equality` is required, see jenkins error log [here](https://riot-ci.inet.haw-hamburg.de/job/RIOT_PR/job/PR-6600/21/artifact/error_native_llvm_pkg_tests.log) for LLVM toolchain